src/arrayMethodReduce.js

```diff
@@ -5,7 +5,20 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let prev = initialValue;
+    let i = 0;
```

Comment:
I think it's better to give the variable a more understandable name.
For example "startIndex"
```suggestion
    let startIndex = 0;
```